modules/images/webp-uploads/helper.php

```diff
@@ -65,6 +65,33 @@ function webp_uploads_get_upload_image_mime_transforms() {
  * @return array|WP_Error An array with the file and filesize if the image was created correctly otherwise a WP_Error
  */
 function webp_uploads_generate_additional_image_source( $attachment_id, array $size_data, $mime, $destination_file_name = null ) {
+
+	/**
+	 * Filter to allow the generation of additional image sources, in which a defined mime type
+	 * can be transformed and create additional mime types for the file.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array  $image         Image data {'path'=>string, 'file'=>string, 'width'=>int, 'height'=>int, 'mime-type'=>string} or null or WP_Error.
```

Comment:
This needs to be `array|WP_Error|null`. When updating, make sure to indent the following param docs correctly.

Reply:
Fixed now!